include/rocksdb/cache.h

```diff
@@ -384,9 +387,7 @@ class Cache {
   // Prerequisite: no entry is referenced.
   virtual void EraseUnRefEntries() = 0;
 
-  virtual std::string GetPrintableOptions() const { return ""; }
```

Comment:
Can we keep this interface even we can call Customizable function to achieve the same?

Reply:
It is already there (via Configurable)